util/QueryHelper.js

```diff
@@ -0,0 +1,40 @@
+class QueryHelper {
+  constructor(Model, query) {
+    this.Model = Model;
+    this.query = query;
+    this.qb = Model.query();
+    this.handlePagination();
+    this.handleSorting();
+    this.handleFields();
+  }
+
+  async handlePagination() {
+    if (this.query.offset) {
+      this.qb.offset(this.query.offset);
+      delete this.query.offset;
+    }
+    if (this.query.limit) {
+      this.qb.limit(this.query.limit);
+      delete this.query.limit;
+    }
+    return this;
+  }
+
+  async handleSorting() {
+    if (this.query.orderBy) {
+      this.qb.orderBy(this.query.orderBy);
+      delete this.query.orderBy;
+    }
+    return this;
+  }
+
+  async handleFields() {
+    if (this.query.fields) {
```

Comment:
should we expose fields to the API ? Tempted to say "no" atm?

Reply:
From the µs API, sure? We can leave it from the user-facing REST API though.